io.github.rimsort.RimSort.metainfo.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<component type="desktop-application">
+  <id>io.github.rimsort.RimSort</id>
+  <url type="homepage">https://rimsort.github.io/RimSort/</url>
+  <url type="vcs-browser">https://github.com/RimSort/RimSort</url>
+
+  <name>RimSort</name>
+  <summary>
+  RimSort is an open source mod manager for RimWorld,alternative to RimPy Mod Manager
+  </summary>
+
+  <metadata_license>MIT</metadata_license>
+  <project_license>AGPL-3.0-only</project_license>
+  <categories>
+      <category>Game</category>
+  </categories>
+  <description>
+    <p>
+      A free and open source multi-platform mod manager for the video game RimWorld. Built from the ground up to be reliable and community managed. Includes support for Linux, Mac, and Windows.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">io.github.rimsort.RimSort.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>
+        https://github.com/RimSort/RimSort/blob/main/docs/rimsort_preview.png?raw=true
```

Comment:
this should never refer to the tip of a branch as it is not immutable.

Reply:
Done thanks